opencollective/opencollective-api
# Save Fees on Top as Transactions

Resolves https://github.com/opencollective/opencollective/issues/3259

### Testing locally
1. Run `dev-20200624-add-transferwise-and-feesOnTop-records` migration.
2. Donate to any given collective not hosted by Open Collective Inc adding a fee on top.
    - Make sure the host of this collective has `settings.feesOnTop: true`
    - Try the same with a collective that has a currency different from 'USD'.
